### PR TITLE
Land surface upgrades for HR2

### PR DESCRIPTION
**Pull request by HelinWei-NOAA**
## Description

Land surface upgrades for HR2. The detail can be found from ccpp-physics [PR #78](https://github.com/ufs-community/ccpp-physics/pull/78)
 
Is a change of answers expected from this PR?  Yes

### Issue(s) addressed

global workflow [issue 1614](https://github.com/NOAA-EMC/global-workflow/issues/1614)
global workflow [issue 1617](https://github.com/NOAA-EMC/global-workflow/issues/1617)

## Testing

How were these changes tested?  
What compilers / HPCs was it tested with?  
Are the changes covered by regression tests? (If not, why? Do new tests need to be added?)  
Have the ufs-weather-model regression test been run? On what platform?  
- Will the code updates change regression test baseline? If yes, why? Please show the baseline directory below.
- Please commit the regression test log files in your ufs-weather-model branch

## Dependencies

If testing this branch requires non-default branches in other repositories, list them.
Those branches should have matching names (ideally)

Do PRs in upstream repositories need to be merged first?
If so add the "waiting for other repos" label and list the upstream PRs
- waiting on noaa-emc/nems/pull/<pr_number>
- waiting on noaa-emc/fv3atm/pull/<pr_number>

### CODE REVIEW

**Comment by BrianCurtis-NOAA** [comment]
This needs reviews before the UFSWM PR can get into the commit queue.

**Comment by DusanJovic-NOAA** [comment]
This branch (HelinWei-NOAA:hr2_land1) is 5 commits behind develop

**Comment by HelinWei-NOAA** [comment]
I will update to the current after I finish updating FV3 (Sam refactored FV3GFS_io.F90 last week).

**Comment by HelinWei-NOAA** [comment]
@DusanJovic-NOAA The code is now up-to-date with the develop.

File: ccpp/suites/suite_FV3_CPT_v0.xml
Comment (DusanJovic-NOAA) [inline ccpp/suites/suite_FV3_CPT_v0.xml:1]: Why are all these files moved from 'suites_not_used' to 'suites' ?

Comment (HelinWei-NOAA) [reply ccpp/suites/suite_FV3_CPT_v0.xml:1]: I tried to merge your last fv3 commit to my branch before I pushed my local change then reverted that merge. So I might mess them up.

Comment (HelinWei-NOAA) [reply ccpp/suites/suite_FV3_CPT_v0.xml:1]: @DusanJovic-NOAA It shows HelinWei-NOAA:hr2_land1 is up to date with all commits from NOAA-EMC:develop. Did I do anything wrong?

Comment (DusanJovic-NOAA) [reply ccpp/suites/suite_FV3_CPT_v0.xml:1]: It is true that HelinWei-NOAA:hr2_land1 is up to date with all commits from NOAA-EMC:develop, meaning hr2_land1 is not behind develop, but that does not mean that there are no additional commits.

Specifically:

In this commit https://github.com/HelinWei-NOAA/fv3atm/commit/8315bcc9ca708231a700adf36141afa969fa6e11 you merged chages from develop, which moved all unused suites to 'suites_not_used' directory.

But then in the following commit : https://github.com/HelinWei-NOAA/fv3atm/commit/f2043f1b45baafb712cd2e070c0b03b26dc22562 you reverted the previous commit. Why?

Comment (HelinWei-NOAA) [reply ccpp/suites/suite_FV3_CPT_v0.xml:1]: I had planed to push my local change to the github first then merge your change over there. I accidentally merged your change first. So I reverted that merge and pushed my local change there. Then I found there was no difference when I tried to merge your change.

**Review by barlage — APPROVED** [review]
Looks good to me. I don't see any issues.

**Comment by FernandoAndrade-NOAA** [comment]
Requesting @BrianCurtis-NOAA @jkbk2004 @Qingfu-Liu  @zach1221 for review and approvals. Apologies, I don't have permissions to add reviewers to this PR.

File: ccpp/suites/suite_FV3_CPT_v0.xml
Comment (FernandoAndrade-NOAA) [reply ccpp/suites/suite_FV3_CPT_v0.xml:1]: @DusanJovic-NOAA  I believe 42417c4 has resolved and moved the suites back to the suites_not_used directory, if you could confirm and mark resolved.

**Review by BrianCurtis-NOAA — APPROVED** [review]

**Review by Qingfu-Liu — APPROVED** [review]

**Comment by grantfirl** [comment]
@HelinWei-NOAA https://github.com/ufs-community/ccpp-physics/pull/78 was merged. Please update the ccpp/physics submodule pointer and revert .gitmodules.

**Comment by HelinWei-NOAA** [comment]
@grantfirl Done.
> @HelinWei-NOAA [ufs-community/ccpp-physics#78](https://github.com/ufs-community/ccpp-physics/pull/78) was merged. Please update the ccpp/physics submodule pointer and revert .gitmodules.

**Comment by FernandoAndrade-NOAA** [comment]
Ok, this PR is ready to continue with merge process @jkbk2004

**Comment by FernandoAndrade-NOAA** [comment]
@jkbk2004 The noahmp [#7](https://github.com/NOAA-EMC/noahmp/pull/7) PR was merged in. I believe this is ready as well

**Pull request merged**